src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -190,16 +216,15 @@ bool emberAfGroupsClusterAddGroupCallback(app::CommandHandler * commandObj, cons
 bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, const app::ConcreteCommandPath & commandPath,
                                            const Commands::ViewGroup::DecodableType & commandData)
 {
-    auto & groupId = commandData.groupId;
-
-    EmberAfStatus status                                          = EMBER_ZCL_STATUS_NOT_FOUND;
-    CHIP_ERROR err                                                = CHIP_NO_ERROR;
-    uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
+    auto fabricIndex = GetFabricIndex(commandObj);
+    auto & groupId   = commandData.groupId;
+    auto endpointId  = commandPath.mEndpointId;
 
-    // Get the group name, if supported
-    emberAfPluginGroupsServerGetGroupNameCallback(emberAfCurrentEndpoint(), groupId, groupName);
+    EmberAfStatus status = EMBER_ZCL_STATUS_NOT_FOUND;
+    CHIP_ERROR err       = CHIP_NO_ERROR;
+    CharSpan groupName;
 
-    emberAfGroupsClusterPrintln("RX: ViewGroup 0x%2x", groupId);
+    emberAfGroupsClusterPrintln("RX: ViewGroup 0x%02x", groupId);
 
     // For all networks, View Group commands can only be addressed to a
     // single device.
```

Comment:
Comments like this that relate to APS should be revisited/removed. This is not Matter terminology here (device/network).